.gitignore

```diff
@@ -84,3 +84,5 @@ megatest.nim
 /outputGotten.txt
 
 /lib/pure/*.js
+
+!/.builds/
```

Comment:
We already have a ``ci`` directory, can we use that instead?

Reply:
Unfortunately builds.sr.ht looks for a `.builds` directory or a `.build.yml` file in the root.